numpy/lib/tests/test_function_base.py

```diff
@@ -1732,6 +1732,15 @@ def test_string_ticket_1892(self):
         s = '0123456789' * 10
         assert_equal(s, f(s))
 
+    def test_dtype_promotion_ticket_29189(self):
```

Comment:
```suggestion
    def test_dtype_promotion_gh_29189(self):
```
(You can't know, but "ticket" refers to the track NumPy was running before github.)

Reply:
Oh didn't know, I just reproduced the pattern I saw in the file.